dev-client/src/screens/SiteNotesScreen/components/SiteInstructionsCard.tsx

```diff
@@ -18,25 +18,26 @@
 import {useCallback} from 'react';
 import {useTranslation} from 'react-i18next';
 
+import {Project} from 'terraso-client-shared/project/projectTypes';
+
 import {Card} from 'terraso-mobile-client/components/Card';
 import {Icon} from 'terraso-mobile-client/components/icons/Icon';
 import {Row, Text} from 'terraso-mobile-client/components/NativeBaseAdapters';
 import {useNavigation} from 'terraso-mobile-client/navigation/hooks/useNavigation';
 
 type Props = {
-  siteInstructions: string;
+  project: Project;
 };
 
-export const SiteInstructionsCard = ({siteInstructions}: Props) => {
+export const SiteInstructionsCard = ({project}: Props) => {
```

Comment:
Wondering if we should rename this (in another ticket??) to like `PinnedNoteCard`; `SiteInstructions` tripped me up a little since it's a project being passed around.

Reply:
Yeah, I was thinking about this too. The `SiteInstructions` name goes all the way to the backend, so was debating how much renaming to do and ended up on no renaming 😅  I'll rename a few things here, and make a tech debt ticket for changes that touch the other repos: https://github.com/techmatters/terraso-product/issues/1156. 

Oo I do like `PinnedNote`, and apparently there's already some backend project action that's called "EDIT_PINNED_NOTE".